Add stale-while-revalidate ads performance case study blog post

thebengeu opened the pull request:
Fixes #1862.

https://deploy-preview-2288--web-dev-staging.netlify.com/ads-case-study-stale-while-revalidate/

Changes proposed in this pull request:

- Add stale-while-revalidate ads performance case study blog post

kaycebasques commented on src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md:
```diff
@@ -0,0 +1,78 @@
+
+## About
+
+The Google Publisher Tag (GPT) is the ad tagging script for Google Ad Manager that requests and renders display ads on the web. By implementing a simple `stale-while-revalidate` HTTP header for GPT, we were able to improve the speed and performance of Google display ads for our publisher partners. This same technique can be applied to many other scenarios where loading scripts as quickly as possi ...
```
```suggestion
The [Google Publisher Tag (GPT)](https://support.google.com/admanager/answer/181073) is the ad tagging script for Google Ad Manager that requests and renders display ads on the web. By implementing a simple [`stale-while-revalidate`](/stale-while-revalidate/) HTTP header for GPT, the GPT team was able to improve the speed and performance of Google display ads for its publisher partners. This same technique can be applied to any other scenario where loading scripts as quickly as possible is more important than loading the freshest code.
```

Changes:

* Linking to explanation of GPT
* Linking to Jeff's stale-while-revalidate article
* Removing the use of "we" because it slightly implies that web.dev == GPT team (it may seem like a nitpick but just take my word that there are bigger content strategy motivations behind this)

That explanation of how this is generally applicable to web developers at the end is GREAT!

kaycebasques commented on src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md:
```diff
@@ -0,0 +1,78 @@
+
+Source: Google Internal Data, June to July 2019.
+
+## About
```
```suggestion
## Background
```

kaycebasques commented on src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md:
```diff
@@ -0,0 +1,78 @@
+
+## The problem
+
+GPT is deployed as a bootstrapping script, `gpt.js`, which is given a short time to live (TTL) of 15 minutes. This short TTL allows the script to be updated or rolled back quickly. Once loaded, gpt.js requests and loads additional implementation scripts, which have a longer TTL.
```
```suggestion
GPT is deployed as a bootstrapping script, `gpt.js`, which is given a short time to live (TTL) of 15 minutes. This short TTL allows the script to be updated or rolled back quickly. Once loaded, `gpt.js` requests and loads additional implementation scripts, which have a longer TTL.
```

`gpt.js` should be formatted as inline code throughout the article

kaycebasques commented on src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md:
```diff
@@ -0,0 +1,78 @@
+
+This setting means that if `gpt.js` is requested between 15 and 60 minutes after the previous cached value, then the cached value will be used to fulfill the request even though it's stale. At the same time, a revalidation request will be made in the background to populate the cache with a fresh value for future use.
+
+Chrome rolled out `stale-while-revalidate` in version 75 to 99% of all traffic, leaving 1% of traffic with the feature disabled temporarily to measure it's impact. The Google display ads team logged metrics from this 1% (the experimental group) as well as a 1% sample of traffic with the feature enabled (the control group), to test the effectiveness of `stale-while-revalidate` for ad scripts. Over ...
```
```suggestion
Chrome rolled out `stale-while-revalidate` in version 75 to 99% of all traffic, leaving 1% of traffic with the feature disabled temporarily to measure its impact. The Google display ads team logged metrics from this 1% (the experimental group) as well as a 1% sample of traffic with the feature enabled (the control group), to test the effectiveness of `stale-while-revalidate` for ad scripts. Over the course of 2 weeks of metrics logged from a sample size of 5.2 billion Google display ad impressions, the control group observed:
```
it's => its

kaycebasques commented on src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md:
```diff
@@ -0,0 +1,78 @@
+
+## Implementing stale-while-revalidate on your site
+
+We have seen that making a relatively simple change to your HTTP headers with `stale-while-revalidate` can improve speed and boost business metrics. Check out the [Keeping things fresh with stale-while-revalidate](/stale-while-revalidate) post for more on implementing `stale-while-revalidate` on your own site.
```
```suggestion
The GPT team has seen that making a relatively simple change to HTTP headers with `stale-while-revalidate` can improve speed and boost business metrics. Check out the [Keeping things fresh with stale-while-revalidate](/stale-while-revalidate) post for more on implementing `stale-while-revalidate` on your own site.
```

Avoiding "we" here for the same reason mentioned above (it's part of a broader content strategy topic the details of which probably aren't necessary here)

thebengeu commented:
Thanks @kaycebasques! I've made all your suggested changes, recreated the chart as a 13KB SVG, and added hero and thumbnail images.

I squashed the commits since it appears that the CLA bot didn't recognize your personal email for your suggested commits. Maybe because of that, it doesn't appear like I can merge this PR myself anymore.

thebengeu commented:
@kaycebasques added a few more tiny changes, this PR should be good to merge.
